### ISSUE TITLE
【EESS】New submissions for Mon,  5 Jun 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#264]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Introduction of Medical Imaging Modalities
 - **Authors:** S. K. M Shadekul Islam, MD Abdullah Al Nasim, Ismail Hossain, Dr. Md Azim Ullah, Dr. Kishor Datta Gupta, Md Monjur Hossain Bhuiyan
 - **Subjects:** Image and Video Processing (eess.IV); Medical Physics (physics.med-ph)
 - **Arxiv link:** https://arxiv.org/abs/2306.01022
 - **Pdf link:** https://arxiv.org/pdf/2306.01022
 - **Abstract**
 The diagnosis and treatment of various diseases had been expedited with the help of medical imaging. Different medical imaging modalities, including X-ray, Computed Tomography (CT), Magnetic Resonance Imaging (MRI), Nuclear Imaging, Ultrasound, Electrical Impedance Tomography (EIT), and Emerging Technologies for in vivo imaging modalities is presented in this chapter, in addition to these modalities, some advanced techniques such as contrast-enhanced MRI, MR approaches for osteoarthritis, Cardiovascular Imaging, and Medical Imaging data mining and search. Despite its important role and potential effectiveness as a diagnostic tool, reading and interpreting medical images by radiologists is often tedious and difficult due to the large heterogeneity of diseases and the limitation of image quality or resolution. Besides the introduction and discussion of the basic principles, typical clinical applications, advantages, and limitations of each modality used in current clinical practice, this chapter also highlights the importance of emerging technologies in medical imaging and the role of data mining and search aiming to support translational clinical research, improve patient care, and increase the efficiency of the healthcare system.
### Deep Learning based Skin-layer Segmentation for Characterizing Cutaneous  Wounds from Optical Coherence Tomography Images
 - **Authors:** Prashant Kumar, Swatantra Dhara, Ayan Gope, Jyotirmoy Chatterjee, Subhamoy Mandal
 - **Subjects:** Image and Video Processing (eess.IV); Medical Physics (physics.med-ph)
 - **Arxiv link:** https://arxiv.org/abs/2306.01252
 - **Pdf link:** https://arxiv.org/pdf/2306.01252
 - **Abstract**
 Optical coherence tomography (OCT) is a medical imaging modality that allows us to probe deeper substructures of skin. The state-of-the-art wound care prediction and monitoring methods are based on visual evaluation and focus on surface information. However, research studies have shown that sub-surface information of the wound is critical for understanding the wound healing progression. This work demonstrated the use of OCT as an effective imaging tool for objective and non-invasive assessments of wound severity, the potential for healing, and healing progress by measuring the optical characteristics of skin components. We have demonstrated the efficacy of OCT in studying wound healing progress in vivo small animal models. Automated analysis of OCT datasets poses multiple challenges, such as limitations in the training dataset size, variation in data distribution induced by uncertainties in sample quality and experiment conditions. We have employed a U-Net-based model for the segmentation of skin layers based on OCT images and to study epithelial and regenerated tissue thickness wound closure dynamics and thus quantify the progression of wound healing. In the experimental evaluation of the OCT skin image datasets, we achieved the objective of skin layer segmentation with an average intersection over union (IOU) of 0.9234. The results have been corroborated using gold-standard histology images and co-validated using inputs from pathologists. Clinical Relevance: To monitor wound healing progression without disrupting the healing procedure by superficial, noninvasive means via the identification of pixel characteristics of individual layers.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
### Differentiable Grey-box Modelling of Phaser Effects using Frame-based  Spectral Processing
 - **Authors:** Alistair Carson, Cassia Valentini-Botinhao, Simon King, Stefan Bilbao
 - **Subjects:** Audio and Speech Processing (eess.AS); Machine Learning (cs.LG); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2306.01332
 - **Pdf link:** https://arxiv.org/pdf/2306.01332
 - **Abstract**
 Machine learning approaches to modelling analog audio effects have seen intensive investigation in recent years, particularly in the context of non-linear time-invariant effects such as guitar amplifiers. For modulation effects such as phasers, however, new challenges emerge due to the presence of the low-frequency oscillator which controls the slowly time-varying nature of the effect. Existing approaches have either required foreknowledge of this control signal, or have been non-causal in implementation. This work presents a differentiable digital signal processing approach to modelling phaser effects in which the underlying control signal and time-varying spectral response of the effect are jointly learned. The proposed model processes audio in short frames to implement a time-varying filter in the frequency domain, with a transfer function based on typical analog phaser circuit topology. We show that the model can be trained to emulate an analog reference device, while retaining interpretable and adjustable parameters. The frame duration is an important hyper-parameter of the proposed model, so an investigation was carried out into its effect on model accuracy. The optimal frame length depends on both the rate and transient decay-time of the target effect, but the frame length can be altered at inference time without a significant change in accuracy.
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### NNMobile-Net: Rethinking CNN Design for Deep Learning-Based Retinopathy  Research
 - **Authors:** Wenhui Zhu, Peijie Qiu, Natasha Lepore, Oana M. Dumitrascu, Yalin Wang
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2306.01289
 - **Pdf link:** https://arxiv.org/pdf/2306.01289
 - **Abstract**
 Retinal diseases (RD) are the leading cause of severe vision loss or blindness. Deep learning-based automated tools play an indispensable role in assisting clinicians in diagnosing and monitoring RD in modern medicine. Recently, an increasing number of works in this field have taken advantage of Vision Transformer to achieve state-of-the-art performance with more parameters and higher model complexity compared to Convolutional Neural Networks (CNNs). Such sophisticated and task-specific model designs, however, are prone to be overfitting and hinder their generalizability. In this work, we argue that a channel-aware and well-calibrated CNN model may overcome these problems. To this end, we empirically studied CNN's macro and micro designs and its training strategies. Based on the investigation, we proposed a no-new-MobleNet (nn-MobileNet) developed for retinal diseases. In our experiments, our generic, simple and efficient model superseded most current state-of-the-art methods on four public datasets for multiple tasks, including diabetic retinopathy grading, fundus multi-disease detection, and diabetic macular edema classification. Our work may provide novel insights into deep learning architecture design and advance retinopathy research.
### Active Noise Control in The New Century: The Role and Prospect of Signal  Processing
 - **Authors:** Dongyuan Shi, Bhan Lam, Woon-Seng Gan, Jordan Cheer
 - **Subjects:** Audio and Speech Processing (eess.AS); Signal Processing (eess.SP); Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2306.01425
 - **Pdf link:** https://arxiv.org/pdf/2306.01425
 - **Abstract**
 Since Paul Leug's 1933 patent application for a system for the active control of sound, the field of active noise control (ANC) has not flourished until the advent of digital signal processors forty years ago. Early theoretical advancements in digital signal processing and processors laid the groundwork for the phenomenal growth of the field, particularly over the past quarter-century. The widespread commercial success of ANC in aircraft cabins, automobile cabins, and headsets demonstrates the immeasurable public health and economic benefits of ANC. This article continues where Elliott and Nelson's 1993 Signal Processing Magazine article and Elliott's 1997 50th anniversary commentary~\cite{kahrs1997past} on ANC left off, tracing the technical developments and applications in ANC spurred by the seminal texts of Nelson and Elliott (1991), Kuo and Morgan (1996), Hansen and Snyder (1996), and Elliott (2001) since the turn of the century. This article focuses on technical developments pertaining to real-world implementations, such as improving algorithmic convergence, reducing system latency, and extending control to non-stationary and/or broadband noise, as well as the commercial transition challenges from analog to digital ANC systems. Finally, open issues and the future of ANC in the era of artificial intelligence are discussed.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Towards Selection of Text-to-speech Data to Augment ASR Training
 - **Authors:** Shuo Liu, Leda Sarı, Chunyang Wu, Gil Keren, Yuan Shangguan, Jay Mahadeokar, Ozlem Kalinli
 - **Subjects:** Audio and Speech Processing (eess.AS); Computation and Language (cs.CL); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2306.00998
 - **Pdf link:** https://arxiv.org/pdf/2306.00998
 - **Abstract**
 This paper presents a method for selecting appropriate synthetic speech samples from a given large text-to-speech (TTS) dataset as supplementary training data for an automatic speech recognition (ASR) model. We trained a neural network, which can be optimised using cross-entropy loss or Arcface loss, to measure the similarity of a synthetic data to real speech. We found that incorporating synthetic samples with considerable dissimilarity to real speech, owing in part to lexical differences, into ASR training is crucial for boosting recognition performance. Experimental results on Librispeech test sets indicate that, in order to maintain the same speech recognition accuracy as when using all TTS data, our proposed solution can reduce the size of the TTS data down below its $30\,\%$, which is superior to several baseline methods.
### Deep Learning based Skin-layer Segmentation for Characterizing Cutaneous  Wounds from Optical Coherence Tomography Images
 - **Authors:** Prashant Kumar, Swatantra Dhara, Ayan Gope, Jyotirmoy Chatterjee, Subhamoy Mandal
 - **Subjects:** Image and Video Processing (eess.IV); Medical Physics (physics.med-ph)
 - **Arxiv link:** https://arxiv.org/abs/2306.01252
 - **Pdf link:** https://arxiv.org/pdf/2306.01252
 - **Abstract**
 Optical coherence tomography (OCT) is a medical imaging modality that allows us to probe deeper substructures of skin. The state-of-the-art wound care prediction and monitoring methods are based on visual evaluation and focus on surface information. However, research studies have shown that sub-surface information of the wound is critical for understanding the wound healing progression. This work demonstrated the use of OCT as an effective imaging tool for objective and non-invasive assessments of wound severity, the potential for healing, and healing progress by measuring the optical characteristics of skin components. We have demonstrated the efficacy of OCT in studying wound healing progress in vivo small animal models. Automated analysis of OCT datasets poses multiple challenges, such as limitations in the training dataset size, variation in data distribution induced by uncertainties in sample quality and experiment conditions. We have employed a U-Net-based model for the segmentation of skin layers based on OCT images and to study epithelial and regenerated tissue thickness wound closure dynamics and thus quantify the progression of wound healing. In the experimental evaluation of the OCT skin image datasets, we achieved the objective of skin layer segmentation with an average intersection over union (IOU) of 0.9234. The results have been corroborated using gold-standard histology images and co-validated using inputs from pathologists. Clinical Relevance: To monitor wound healing progression without disrupting the healing procedure by superficial, noninvasive means via the identification of pixel characteristics of individual layers.
### Zero-Shot Blind Audio Bandwidth Extension
 - **Authors:** Eloi Moliner, Filip Elvander, Vesa Välimäki
 - **Subjects:** Audio and Speech Processing (eess.AS); Machine Learning (cs.LG); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2306.01433
 - **Pdf link:** https://arxiv.org/pdf/2306.01433
 - **Abstract**
 Audio bandwidth extension involves the realistic reconstruction of high-frequency spectra from bandlimited observations. In cases where the lowpass degradation is unknown, such as in restoring historical audio recordings, this becomes a blind problem. This paper introduces a novel method called BABE (Blind Audio Bandwidth Extension) that addresses the blind problem in a zero-shot setting, leveraging the generative priors of a pre-trained unconditional diffusion model. During the inference process, BABE utilizes a generalized version of diffusion posterior sampling, where the degradation operator is unknown but parametrized and inferred iteratively. The performance of the proposed method is evaluated using objective and subjective metrics, and the results show that BABE surpasses state-of-the-art blind bandwidth extension baselines and achieves competitive performance compared to non-blind filter-informed methods when tested with synthetic data. Moreover, BABE exhibits robust generalization capabilities when enhancing real historical recordings, effectively reconstructing the missing high-frequency content while maintaining coherence with the original recording. Subjective preference tests confirm that BABE significantly improves the audio quality of historical music recordings. Examples of historical recordings restored with the proposed method are available on the companion webpage: (this http URL)
### On Crowdsourcing-design with Comparison Category Rating for Evaluating  Speech Enhancement Algorithms
 - **Authors:** Angélica S. Z. Suárez, Clément Laroche, Line H. Clemmensen, Sneha Das
 - **Subjects:** Audio and Speech Processing (eess.AS)
 - **Arxiv link:** https://arxiv.org/abs/2306.01538
 - **Pdf link:** https://arxiv.org/pdf/2306.01538
 - **Abstract**
 Speech enhancement techniques improve the quality or the intelligibility of an audio signal by removing unwanted noise. It is used as preprocessing in numerous applications such as speech recognition, hearing aids, broadcasting and telephony. The evaluation of such algorithms often relies on reference-based objective metrics that are shown to correlate poorly with human perception. In order to evaluate audio quality as perceived by human observers it is thus fundamental to resort to subjective quality assessment. In this paper, a user evaluation based on crowdsourcing (subjective) and the Comparison Category Rating (CCR) method is compared against the DNSMOS, ViSQOL and 3QUEST (objective) metrics. The overall quality scores of three speech enhancement algorithms from real time communications (RTC) are used in the comparison using the P.808 toolkit. Results indicate that while the CCR scale allows participants to identify differences between processed and unprocessed audio samples, two groups of preferences emerge: some users rate positively by focusing on noise suppression processing, while others rate negatively by focusing mainly on speech quality. We further present results on the parameters, size considerations and speaker variations that are critical and should be considered when designing the CCR-based crowdsourcing evaluation.
### Detecting Low Pass Graph Signals via Spectral Pattern: Sampling  Complexity and Applications
 - **Authors:** Chenyue Zhang, Yiran He, Hoi-To Wai
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2306.01553
 - **Pdf link:** https://arxiv.org/pdf/2306.01553
 - **Abstract**
 This paper proposes a blind detection problem for low pass graph signals. Without assuming knowledge of the graph topology in advance, we aim to detect if a set of graph signal observations are generated from a low pass graph filter. Our problem is motivated by the widely adopted assumption of low pass (a.k.a.~smooth) signals required by many existing works in graph signal processing (GSP), as well as the longstanding problem of network dynamics identification. Focusing on detecting low pass graph signals whose cutoff frequency coincides with the number of clusters present, our key idea is to develop blind detector leveraging the unique spectral pattern exhibited by low pass graph signals. We analyze the sample complexity of these detectors considering the effects of graph filter's properties, random delays. We show novel applications of the blind detector on robustifying graph learning, identifying antagonistic ties in opinion dynamics, and detecting anomalies in power systems. Numerical experiments validate our findings.
### An Attentive-based Generative Model for Medical Image Synthesis
 - **Authors:** Jiayuan Wang, Q. M. Jonathan Wu, Farhad Farhad
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2306.01562
 - **Pdf link:** https://arxiv.org/pdf/2306.01562
 - **Abstract**
 Magnetic resonance (MR) and computer tomography (CT) imaging are valuable tools for diagnosing diseases and planning treatment. However, limitations such as radiation exposure and cost can restrict access to certain imaging modalities. To address this issue, medical image synthesis can generate one modality from another, but many existing models struggle with high-quality image synthesis when multiple slices are present in the dataset. This study proposes an attention-based dual contrast generative model, called ADC-cycleGAN, which can synthesize medical images from unpaired data with multiple slices. The model integrates a dual contrast loss term with the CycleGAN loss to ensure that the synthesized images are distinguishable from the source domain. Additionally, an attention mechanism is incorporated into the generators to extract informative features from both channel and spatial domains. To improve performance when dealing with multiple slices, the $K$-means algorithm is used to cluster the dataset into $K$ groups, and each group is used to train a separate ADC-cycleGAN. Experimental results demonstrate that the proposed ADC-cycleGAN model produces comparable samples to other state-of-the-art generative models, achieving the highest PSNR and SSIM values of 19.04385 and 0.68551, respectively. We publish the code at https://github.com/JiayuanWang-JW/ADC-cycleGAN.
### A Conditional Normalizing Flow for Accelerated Multi-Coil MR Imaging
 - **Authors:** Jeffrey Wen, Rizwan Ahmad, Philip Schniter
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2306.01630
 - **Pdf link:** https://arxiv.org/pdf/2306.01630
 - **Abstract**
 Accelerated magnetic resonance (MR) imaging attempts to reduce acquisition time by collecting data below the Nyquist rate. As an ill-posed inverse problem, many plausible solutions exist, yet the majority of deep learning approaches generate only a single solution. We instead focus on sampling from the posterior distribution, which provides more comprehensive information for downstream inference tasks. To do this, we design a novel conditional normalizing flow (CNF) that infers the signal component in the measurement operator's nullspace, which is later combined with measured data to form complete images. Using fastMRI brain and knee data, we demonstrate fast inference and accuracy that surpasses recent posterior sampling techniques for MRI. Code is available at https://github.com/jwen307/mri_cnf/
